src/components/AppBar.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, AppBar, Toolbar, Typography } from '@mui/material'
+import React from 'react'
+
+interface Props {
+
+}
```

Comment:
Remove empty `Props`

Do cleanup before final push and do not add anything which is not in use